packages/ingest-mongodb-public/src/sources/mongodb-university/makeUniversityPages.test.ts

```diff
@@ -89,3 +90,53 @@ describe("convertVideoTranscriptFromSrtToTxt()", () => {
     expect(convertedTranscript).toEqual(txtTranscript);
   });
 });
+
+describe("generateContentDescriptionMarkdown()", () => {
+  const expected_markdown = `# Atlas Search
+This course is designed to help you use Atlas Search in your applications for a seamless full-text search experience. You’ll learn how to write Atlas Search queries and manage search indexes by using the Atlas UI, CLI, and MongoDB Shell. Then you’ll build on this foundation by writing advanced search queries that use the compound operator, nesting, and features such as fuzzy search and autocomplete. 
```

Comment:
```suggestion
  const expected_markdown = `# Atlas Search

This course is designed to help you use Atlas Search in your applications for a seamless full-text search experience. You’ll learn how to write Atlas Search queries and manage search indexes by using the Atlas UI, CLI, and MongoDB Shell. Then you’ll build on this foundation by writing advanced search queries that use the compound operator, nesting, and features such as fuzzy search and autocomplete. 
```

add line breaks between titles and subsequent text. do same for all H2